schema/object_identification.js

```diff
@@ -117,7 +118,11 @@ const NodeField = {
   resolve: (root, { __id }, request, rootValue) => {
     const { type, id } = fromGlobalId(__id)
     if (isSupportedType(type)) {
-      const { resolve } = SupportedTypes.typeModules[type]
+      let exported = SupportedTypes.typeModules[type]
```

Comment:
This is because the default export of `filterArtworks` is a method, not the object with a resolve itself.